lang/en.json

```diff
@@ -10,15 +10,15 @@
         "init": "Extracting project source...",
         "success": "Completed project source extraction.",
         "errors": {
-          "write": "An error occured writing temp project source.",
-          "extract": "An error occured extracting project source."
+          "write": "An error occurred writing temp project source.",
+          "extract": "An error occurred extracting project source."
```

Comment:
I don't even think this is the first time we've spelled occurred incorrectly lol